patches/object.values+1.1.1.patch

```diff
@@ -0,0 +1,23 @@
+diff --git a/node_modules/object.values/index.js b/node_modules/object.values/index.js
+index b8ba091..2dc8083 100644
+--- a/node_modules/object.values/index.js
++++ b/node_modules/object.values/index.js
+@@ -1,17 +1,3 @@
+ 'use strict';
+ 
+-var define = require('define-properties');
+-
+-var implementation = require('./implementation');
+-var getPolyfill = require('./polyfill');
+-var shim = require('./shim');
+-
+-var polyfill = getPolyfill();
+-
+-define(polyfill, {
+-	getPolyfill: getPolyfill,
+-	implementation: implementation,
+-	shim: shim
+-});
+-
+-module.exports = polyfill;
++module.exports = Object.values;
```

Comment:
The polyfill was assigning a bunch of garbage to `Object.values`. It's no longer 1999, so I don't think we need this nonsense.